sae_lens/training/activations_store.py

```diff
@@ -22,6 +21,30 @@
 from sae_lens.sae import SAE
 from sae_lens.tokenization_and_batching import concat_and_batch_sequences
 
+torch_to_numpy_dtype_dict: dict[torch.dtype, Any] = {
```

Comment:
It looks like `bfloat16` is missing from this list. I don't think numpy supports bfloat16 though, even though it's a pretty common format for neural net training. Probably the safest would be to map it to `np.float32`, although it's sort of sad to need to use double the space for the numpy mmap in that case.

Reply:
noice